doc/govuk-components.md

```diff
@@ -1,124 +0,0 @@
-# GOV.UK components
-
-Shared partials that encapsulate the HTML/CSS/JS for a common UI component.
-The component files are centrally hosted on static and exposed to applications via [alphagov/slimmer](https://github.com/alphagov/slimmer).
-The partial is exposed over the network, and the CSS/JS are included by the shared templated layout.
-
-The available components and their documentation are exposed by an API at `/templates/govuk_component/docs`, which is consumed by
-[alphagov/govuk_component_guide](https://github.com/alphagov/govuk_component_guide) to generate a living styleguide for components.
-
-* a [Partial View](../app/views/govuk_component) - The template logic and markup, also defines the arguments expected
-* a [SCSS module](../app/assets/stylesheets/govuk-component) - The styling of the component
-* a [Javascript module](../app/assets/javascripts/govuk-component) - JS behaviour of the component
-* [Documentation](../app/views/govuk_component/docs) - a `.yml` per component, describing the component and containing fixture data.
-* a [unit test](../test/govuk_component) - testing the component renders correctly based on parameters passed in
-
-## Principles behind components
-
-### Encapsulate common UI
```

Comment:
@fofr is this transferred or have an equivalent in our new docs? this is pretty good stuff...

Reply:
Yeah, I thought the same. We should transfer to gem.

Reply:
https://github.com/alphagov/govuk_publishing_components/issues/46